src/components/footer/arrowDownSimple/arrow-down-simple.tsx

```diff
@@ -1,20 +1,6 @@
-import styles from "./arrow-down-simple.module.scss";
-import arrowDownSimple from "../../../assets/images/arrowDownSimple.png";
+import React from "react";
+import { ReactComponent as ArrowDownSimpleIcon } from "../../../assets/images/arrowDownSimple.svg";
 
-interface ArrowDownSimpleProps {
-  task: () => void;
-  id: string;
-}
-
-export const ArrowDownSimple: React.FC<ArrowDownSimpleProps> = ({
-  task,
-  id,
-}: Function) => {
-  return (
-    <img
-      className={styles.img}
-      src={arrowDownSimple}
-      onClick={() => task(id)}
-    ></img>
-  );
+export const ArrowDownSimpleComponent = () => {
+  return <ArrowDownSimpleIcon />;
```

Comment:
We don't need a component that returns another component, why would we do this it's redundant. Just use <ArrowDownSimpleIcon> in parent component, instead of using <ArrowDownSimpleComponent> which returns the same

Reply:
done